app/views/layouts/base.html.slim

```diff
@@ -14,5 +14,5 @@ html lang="en-US"
     = yield :styles
 
   body
-    = render "application/alerts"
+    = content_for?(:alerts) ? render(inline: '') : render('application/alerts')
```

Comment:
Does `= render('application/alerts') if content_for?(:alerts)` work? I'm not entirely sure if slim will like it, I don't have much experience with it. If not

```ruby
- if content_for?(:alerts)
  = render('application/alerts')
```
Definitely should